### ISSUE TITLE
training/validation loss difference

### ISSUE THREAD

**Issue by mgrankin** [tloen/alpaca-lora#109]
I'm doing one epoch fine-tuning on my dataset. I observe that the training loss is consistently lower than validation loss. Given that I use default LORA dropout 0.05 and train for just one epoch, it appears to me it should be the opposite. The validation loss should be lower than training loss because the validation inference doesn't apply dropout. 

I've checked with fine-tuning on alpaca data and got similar result during the first epoch of training.  

How is that possible?

<img width="461" alt="Screenshot 2023-03-20 at 18 50 02" src="https://user-images.githubusercontent.com/3540879/226575745-de48c12d-fbee-4190-adee-6c08e4456015.png">

**Comment by mgrankin** [comment]
This is the output of the `finetune.py` with the default dataset. I made a single change and set `eval_steps=20`, instead of 200 to have a better look. 
<img width="1024" alt="Screenshot 2023-03-21 at 18 35 26" src="https://user-images.githubusercontent.com/3540879/226713573-71803a0b-3bf3-4775-8e83-6c3b21e4327a.png">

**Comment by Tongcheng** [comment]
Question @mgrankin , do you know how you were able to produce both training & validation set loss? It seems I can only produce one of it
